app/models/claim.rb

```diff
@@ -471,6 +477,14 @@ def attributes_flagged_by_risk_indicator
 
   private
 
+  def policy_specific_approvable?
+    return true if policy.nil?
+    return true if eligibility.nil?
```

Comment:
Can these ever be `nil` or is this just for the test suite?

Reply:
theres actually no coverage for this, i was just erring on the side of caution cause i have no idea how upstream behaves and didn't want to cause an issue downstream